### ISSUE TITLE
ingest SIDER for hackathon for KS

### ISSUE THREAD

**Issue by realmarcin** [NCATS-Tangerine/translator-knowledge-graph#3]

**Comment by andrewsu** [comment]
FYI, looks like [mychem.info](mychem.info) ingests SIDER.  Example query:

http://mychem.info/v1/chem/KWHRDNMACVLHCE-UHFFFAOYSA-N

```
{
  "_id": "KWHRDNMACVLHCE-UHFFFAOYSA-N",
  "_version": 1,
  "pubchem": {
    ...
  },
  "sider": [
    {
      "_license": "https://goo.gl/8b7ZCQ",
      "meddra": {
        "type": "LLT",
        "umls_id": "C0042029"
      },
      "side_effect": {
        "frequency": "0.448%",
        "name": "Urinary tract infection",
        "placebo": false
      },
      "stitch": {
        "flat": "CID116156130",
        "stereo": "CID016157882"
      }
    },
    {
      "_license": "https://goo.gl/8b7ZCQ",
      "meddra": {
        "type": "PT",
        "umls_id": "C0041912"
      },
      "side_effect": {
        "frequency": "9.04%",
        "name": "Upper respiratory tract infection",
        "placebo": false
      },
      "stitch": {
        "flat": "CID116156130",
        "stereo": "CID016157882"
      }
    },
    ...
}
```

To query by drugname: http://mychem.info/v1/query?q=drugbank.name:imatinib
To query by drugbank ID: http://mychem.info/v1/query?q=drugbank.drugbank_id:DB02968

**Comment by cmungall** [comment]
Nice, I think the mapping to ontology terms should be straightforward. @pnrobinson or @drseb is there anything of your work we can reuse here?

**Comment by pnrobinson** [comment]
?SIDER is using UMLS, which is mapped to HPO. We had our own mapping years ago, but I would suggest just using and evaluating the UMLS to HPO mapping.  Last time I looked, about 5% of the data in SIDR was erroneous, and so additional curation might be needed for medical use.

-Peter



Peter Robinson
Professor and Donald A. Roux Chair, Genomics and Computational Biology
The Jackson Laboratory for Genomic Medicine
860.837.2095 t | 860.990.3130 m | peter.robinson@jax.org | https://robinsongroup.github.io/
Peter Robinson
________________________________

**Comment by cmungall** [comment]
> I would suggest just using and evaluating the UMLS to HPO mapping

good point. We can also use UMLS to MONDO, since some of these will more clearly map to disease entities?

> Last time I looked, about 5% of the data in SIDR was erroneous, and so additional curation might be needed for medical use

I believe the use case here will tolerate this as it's statistical and no one will actually be making clinical recommendations. But it is good to know these stats, build them in as priors etc
